src/components/elements/accordion.tsx

```diff
@@ -159,36 +159,38 @@ const ItemExpanded = ({ is_default_open, child, child_idx, nodes, id }: ItemExpa
 
     return (
         <>
-            <div
-                key={child_idx}
-                style={child.props.parent_style}
-                id={id}
-                ref={(div) => {
-                    nodes[child_idx] = { ref: div }
-                }}
-            >
-                <AccordionWrapper>
-                    <AccordionHeader
-                        onClick={() => setExpanded(!is_expanded)}
-                        role="button"
-                        aria-expanded={is_expanded}
-                        style={child.props.header_style}
-                    >
-                        <Text weight="bold">{child.props.header}</Text>
-                        <div>{child.props.plus ? deployer : current_arrow}</div>
-                    </AccordionHeader>
-                    <div
-                        style={{
-                            overflow: 'hidden',
-                            transition: `height ${TRANSITION_DURATION}ms ease`,
-                            height,
-                            ...child.props.content_style,
-                        }}
-                    >
-                        {child}
-                    </div>
-                </AccordionWrapper>
-            </div>
+            {child.props.is_showed != false && (
```

Comment:
If `child.props.is_showed` will have no properties, it'll false by default. Are we need to extra check?
What do you think by using `!child.props.is_showed` instead?